docs/topic_guides/evaluation/edge_metrics.md

```diff
@@ -1,24 +1,27 @@
 This guide is intended to be a reference guide for Edge Metrics used throughout Splink. It will build up from basic principles into more complex metrics.
 
+!!! note
+    All of these metrics are dependent on having a "ground truth" to compare against. This is generally provided by **clerical labelling** (i.e. labels created by a human). For more on how to generate this ground truth (and the impact that can have on Edge Metrics), check out the [Clerical Labelling Topic Guide](./labelling.md).
+
 ## The Basics
 
 Any Edge (Link) within a Splink model will fall into one of four categories:
 
-#### 1. True Positive
+#### True Positive
 
 A True Positive is a case where a Splink model correctly identifies a match between two records.
 
-#### 2. True Negative
+#### True Negative
 
 A True Negative is a case where a Splink model correctly identifies a non-match between two records.
 
-#### 3. False Positive
+#### False Positive
 
 A False Positive is a case where a Splink model incorrectly predicts a match between two records, when they are actually a non-match.
 
 Also referred to as a Type I Error.
 
-#### 4. False Negative
+#### False Negative
```

Comment:
This is often called a 'missed link' in the literature (e.g. by people like James Doidge see e.g [here](https://www.gov.uk/government/publications/joined-up-data-in-government-the-future-of-data-linking-methods/quality-assessment-in-data-linkage).  I think this is quite useful terminology (or at least, it makes my brain hurt less)